src/core/Suite.js

```diff
@@ -72,6 +74,14 @@ getJasmineRequireObj().Suite = function(j$) {
     this.afterAllFns.unshift(fn);
   };
 
+  Suite.prototype.startTimer = function() {
+    this.timer.start();
+  };
+
+  Suite.prototype.endTimer = function() {
+    this.result.duration = this.timer.elapsed();
```

Comment:
Please add `duration` to the `this.result` documentation comments (line 22) to generate the full API documentation